app/src/main/res/values/strings.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <string name="about_description">Keep track of which episodes you\'ve watched of your favourite TV shows</string>
-  <string name="about_license">Episodes is free software licensed under the GNU GPL (version 3 or later). Please file bugs, write code, or do anything else you can to help!</string>
+  <string name="about_description">Keep track of which episodes you\'ve watched of your favourite shows</string>
+  <string name="about_license">Episodes is copylefted libre software licensed GPLv3+. Please file bugs, write code, translate or do anything else you can to help!</string>
```

Comment:
```suggestion
  <string name="about_license">Episodes is libre software licensed GPLv3+. Please file bugs, write code, translate, or do anything else you can to help!</string>
```
I think "copylefted libre" might be redundant since - to me at least - they imply the same thing. Maybe we pick one?

Reply:
They are actually separate concepts. "Copyleft" means the terms apply to forks of the project. "Libre" is the actual terms meaning use, see, change and share as you want. :)

Reply:
Fair enough! 🙂